cps/web.py

```diff
@@ -47,12 +47,20 @@
 from shutil import move, copyfile
 from tornado.ioloop import IOLoop
 from tornado import version as tornadoVersion
+#from builtins import str
+
+try:
+    from urllib.parse import quote   
+    from imp import reload
+    from past.builtins import xrange
+except:
```

Comment:
To catch any exception use the ```except Exception```, instead of ```except BaseException``` or just ```except```.
Indicate the most specific exception type in ```except``` block.
For example:
```python
# Bad
try:
    mapping[key]
except Exception:
    ...

# Better
try:
    mapping[key]
except KeyError:
    ...
```